### PR TITLE
Update announcements page

### PR DESCRIPTION
**Pull request by akeller**
## What is the purpose of the change

Partially addresses #1801.

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
@felix-mueller can you confirm these dates are correct for release date and EOM? Is there any language we need to add that's different from C7?

File: docs/reference/announcements.md [hunk 1]
```diff
@@ -4,14 +4,32 @@ title: "Announcements"
...
 ---
 
-## Upgrading to 8.2
+## Camunda Platform 8.2
+
+Release date: 11th of April 2023
+
+End of maintenance: 8th of October 2024
+
+### Updating to 8.2
```
Comment (felix-mueller) [inline docs/reference/announcements.md:13]: shouldnt this be part of update guide instead?

File: docs/reference/announcements.md [hunk 2]
```diff
@@ -4,14 +4,32 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
-## Upgrading to 8.2
+## Camunda Platform 8.2
+
+Release date: 11th of April 2023
+
+End of maintenance: 8th of October 2024
```
Comment (felix-mueller) [inline docs/reference/announcements.md:11]: date is correct

**Review by felix-mueller — APPROVED** [review]

File: docs/reference/announcements.md (hunk 1 above)
Comment (akeller) [reply docs/reference/announcements.md:13]: Good catch! I'll move it 😄

**Comment by akeller** [comment]
Depending on how this draft PR looks, I may just close it and open a new one.

**Pull request closed without merging**